src/Sulu/Bundle/AdminBundle/FormMetadata/FormXmlLoader.php

```diff
@@ -101,6 +101,7 @@ protected function parse($resource, \DOMXPath $xpath, $type): LocalizedFormMetad
     private function mapFormsMetadata(ExternalFormMetadata $formMetadata, string $locale): FormMetadata
     {
         $form = new FormMetadata();
+        $form->setTags($this->formMetadataMapper->mapTags($formMetadata->getTags()));
```

Comment:
i cannot set it in the tests because the xml does not accept it - should i remove the mapping here?

Reply:
Why not adding it to the other XML as well? :slightly_smiling_face:

Reply:
will do it